requestmanager/requestmanager_test.go

```diff
@@ -212,9 +212,63 @@ func TestCancelRequestInProgress(t *testing.T) {
 
 	errors := testutil.CollectErrors(requestCtx, t, returnedErrorChan1)
 	require.Len(t, errors, 1)
-	_, ok := errors[0].(graphsync.RequestContextCancelledErr)
+	_, ok := errors[0].(graphsync.RequestClientCancelledErr)
 	require.True(t, ok)
 }
+func TestCancelRequestImperativeNoMoreBlocks(t *testing.T) {
+	ctx := context.Background()
+	td := newTestData(ctx, t)
+	requestCtx, cancel := context.WithTimeout(ctx, time.Second)
+	defer cancel()
```

Comment:
```suggestion
	t.Cleanup(cancel)
```
Any specific reason for using `defer` over `t.Cleanup`?
Ditto in other places

Reply:
wow, cause apparently I never heard of t.Cleanup? Oops. um, I'm going to delay and then let's submit a style PR to fix this across the repo cause it's probably everywhere.